spec/routing/devices_routes_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe "User devices routes", type: :routing do
+  it "renders the user devices routes if the mobile_notifications feature flag is enabled" do
```

Comment:
We recently started being more consistent about using the `:aggregate_failures` metadata in examples  with more than one `assert`, maybe we should this here too:

```suggestion
  it "renders the user devices routes if the mobile_notifications feature flag is enabled", :aggregate_failures do
```